### DISCUSSION TITLE
Can we expose an entire application as remote rather than components?

### DISCUSSION THREAD

**Discussion by AsuraKev** [module-federation/module-federation-examples#4273] [General]
Hi there,

So far the example usage of module federation is around exposing components as remote, I am wondering if we have a full blown react application with react router + state management can that be expose as a remote?

thanks! :)

**Comment by ScriptedAlchemy** [comment]
Yes, you can. They're just modules. An entire app is just a large component. It doesn't matter to the bundler as long as it can be imported.

Our documentation includes framework bridges that help with this as well.
